app/lib/actions/remote_execution/run_hosts_job.rb

```diff
@@ -8,6 +8,8 @@ class RunHostsJob < Actions::ActionWithSubPlans
       middleware.use Actions::Middleware::BindJobInvocation
       middleware.use Actions::Middleware::RecurringLogic
 
+      execution_plan_hooks.use ::Hooks::TriggerRepeat
```

Comment:
SInce the `Actions::Middleware::RecurringLogic` middleware and `::Hooks::TriggerRepeat` are tied together, I wonder if we shouldn't provide a module to be included into action from foreman-tasks, that would register this middleware and hook?